lib/src/ruby/ruby.cc

```diff
@@ -1,11 +1,39 @@
 #include <facter/ruby/ruby.hpp>
+#include <facter/logging/logging.hpp>
 #include <internal/ruby/api.hpp>
 #include <internal/ruby/module.hpp>
 
 using namespace std;
 using namespace facter::facts;
 
+static const string load_puppet = "\
```

Comment:
This could just be a `const char*` since we're only passing it directly to Ruby anyway.